src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.nav {
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    margin-right: 20px;
+    text-transform: uppercase;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    position: relative;
+    height: 60px;
+
+    text-decoration: none;
+    color: $main-accent;
+
+    &:hover {
+      color: $blue-accent;
+    }
+  }
+}
+
+.is-active {
```

Comment:
I understand that `.is-active` is a class required in tests, but you can also add a correct  BEM modifier and move styles there

Because technically `is-active` is a separate BEM block